README.md

```diff
@@ -82,6 +82,138 @@ pip install CMD-V(to paste the copied URL)
 | en_ner_bc5cdr_md |  A spaCy NER model trained on the BC5CDR corpus. | [Download](https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.0/en_ner_bc5cdr_md-0.2.0.tar.gz)|
 | en_ner_bionlp13cg_md |  A spaCy NER model trained on the BIONLP13CG corpus. |[Download](https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.0/en_ner_bionlp13cg_md-0.2.0.tar.gz)|
 
+
+## Additional Pipeline Components
+
+
+### AbbreviationDetector
+The AbbreviationDetector is a Spacy component which implements the abbreviation detection algorithm in "A simple algorithm
+    for identifying abbreviation definitions in biomedical text.", (Schwartz & Hearst, 2003).
+
+You can access the list of abbreviations via the `doc._.abbreviations` attribute and for a given abbreviation,
+you can access it's long form (which is a `spacy.tokens.Span`) using `span._.long_form`, which will point to
+another span in the document.
+
+
+#### Example Usage
+
+```python
+import spacy
+
+from scispacy.abbreviation import AbbreviationDetector
+
+nlp = spacy.load("en_core_sci_sm")
+
+# Add the abbreviation pipe to the spacy pipeline.
+abbreviation_pipe = AbbreviationDetector(nlp)
+nlp.add_pipe(abbreviation_pipe)
+
+doc = nlp("Spinal and bulbar muscular atrophy (SBMA) is an \
+           inherited motor neuron disease caused by the expansion \
+           of a polyglutamine tract within the androgen receptor (AR). \
+           SBMA can be caused by this easily.")
+
+print("Abbreviation", "\t", "Definition")
+for abrv in doc._.abbreviations:
+	print(f"{abrv} \t ({abrv.start}, {abrv.end}) {abrv._.long_form}")
+
+>>> Abbreviation	 Span	    Definition
+>>> SBMA 		 (33, 34)   Spinal and bulbar muscular atrophy
+>>> SBMA 	   	 (6, 7)     Spinal and bulbar muscular atrophy
+>>> AR   		 (29, 30)   androgen receptor
+```
+### UmlsEntityLinker (Alpha feature)
```

Comment:
Could you add a few numbers about the recall of the candidate generation? Just to give a sense of how someone might currently be able to use the linker

Reply:
I'll add this later because it involves re-writing the linking script to remove all the crud which is now nicely re-written in scispacy

Reply:
:+1: